runtime/drivers/clickhouse/clickhouse.go

```diff
@@ -44,6 +44,16 @@ var spec = drivers.Spec{
 			Default:     "false",
 			NoPrompt:    true,
 		},
+		{
+			Key:         "mutable",
```

Comment:
Would prefer to use term `readonly` instead of mutable. It clears the meaning more clearly and also a term used by clickhouse already.

Reply:
What about `mode: readwrite`